hilti/toolchain/src/compiler/global-optimizer.cc

```diff
@@ -23,10 +24,15 @@
 
 namespace hilti {
 
+std::ostream& operator<<(std::ostream& stream, const GlobalOptimizer::ID& x) { return stream << x.second; }
+
 using ModuleID = GlobalOptimizer::ModuleID;
 using StructID = GlobalOptimizer::StructID;
 using FieldID = GlobalOptimizer::FieldID;
 
+using position_t =
+    visitor::Position<hilti::detail::visitor::Iterator<hilti::Node, hilti::detail::visitor::Order::Pre, false>::E>;
+
```

Comment:
Can we avoid this typedef and use the `position_t` as it's available within `Visitor`?

Reply:
That was a leftover from a version where I needed a `position_t` outside of the visitor, removing it now.